lib/oci_utils/migrate/__init__.py

```diff
@@ -0,0 +1,174 @@
+# oci-utils
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown
+# at http://oss.oracle.com/licenses/upl.
+
+import os
+import sys
+import termios
+import tty
+import yaml
+
+from oci_utils.migrate.exception import OciMigrateException
+
+__ocimigrateconffile = '/etc/oci-utils/oci-migrate-conf.yaml'
```

Comment:
User code should stay away from double _ also, please use the std python naming conventions for names ie, this should be:  `_oci_migrate_conf_file`